.travis.yml

```diff
@@ -16,42 +13,23 @@ before_install:
   - go get -u gopkg.in/alecthomas/gometalinter.v2
   - go get -u golang.org/x/lint/golint
 
-# Rename directories so that we're building in $HOME/gopath/src/github.com/watson-developer-cloud/go-sdk
-# When the go-sdk repo is moved to the watson-developer-cloud org on github.com, we can remove these steps
-  - env | grep GO
-  - pwd
-  - mkdir -p $GOPATH/src/github.com/watson-developer-cloud
-  - mv $GOPATH/src/github.ibm.com/arf/go-sdk $GOPATH/src/github.com/watson-developer-cloud
-  - cd $GOPATH/src/github.com/watson-developer-cloud/go-sdk
-  - export TRAVIS_BUILD_DIR=$HOME/gopath/src/github.com/watson-developer-cloud/go-sdk
-  - pwd
-
-
-# Skip the install step. Don't `go get` dependencies. Only build with the code
-# in vendor/
 install:
   - dep ensure
   - 'if [ "${TRAVIS_TAG}" = "${TRAVIS_BRANCH}" ]; then cd appscan; make asoc-tool; cd ../; fi'
-# TODO Change URLs for the repo before we are public.
-#  - git clone --depth=1 --branch=$TRAVIS_BRANCH git@github.ibm.com:arf/go-sdk.git $GOPATH/src/github.com/watson-developer-cloud/go-sdk
+  - git clone --depth=1 --branch=$TRAVIS_BRANCH git@github.com:watson-developer-cloud/go-sdk.git $GOPATH/src/github.com/watson-developer-cloud/go-sdk
```

Comment:
I don't think we need to do an explicit clone of our repo any more, now that we're on github.com.   When the travis build is started, I'm pretty sure it will do a clone of our repo first before it even starts running our script.

Reply:
I think we are using this for our appscan unless this is where we are downloading the SDK to 

https://github.com/watson-developer-cloud/go-sdk/blob/master/appscan/Makefile#L24